cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -1800,4 +1802,31 @@ protected VolumeMount createTempDirVolumeMount() {
     protected VolumeMount createTempDirVolumeMount(String volumeName) {
         return VolumeUtils.createVolumeMount(volumeName, STRIMZI_TMP_DIRECTORY_DEFAULT_MOUNT_PATH);
     }
+
+    /**
+     * Creates a ClusterRoleBinding which is used to bind the SA to the strimzi-kafka-client ClusterRole.
+     *
+     * @param name Name of the cluster role binding.
+     *
+     * @return The cluster role binding.
+     */
+    public ClusterRoleBinding generateKafkaClientClusterRoleBinding(String name) {
+        if (name == null || name.trim().isEmpty()) {
+            return null;
+        }
+
+        Subject subject = new SubjectBuilder()
+                .withKind("ServiceAccount")
+                .withName(getServiceAccountName())
+                .withNamespace(namespace)
+                .build();
+
+        RoleRef roleRef = new RoleRefBuilder()
+                .withName("strimzi-kafka-client")
+                .withApiGroup("rbac.authorization.k8s.io")
+                .withKind("ClusterRole")
+                .build();
+
+        return getClusterRoleBinding(name, subject, roleRef);
+    }
```

Comment:
This abstraction is broken. I don't think it should be there since not everything inheriting from here has anything to do with "client" cluster role bindings.

Reply:
OK, thanks.